utils/ssh.py

```diff
@@ -262,30 +299,38 @@ def cpu_spike(self, seconds=60, cpus=2, **kwargs):
             "for ((i=0; i<instances; i++)) do while (($(date +%s) < $endtime)); "
             "do :; done & done".format(seconds, cpus), **kwargs)
 
-    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT):
-        logger.info("Running rails command `%s`", command)
-        return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {}'.format(command),
-            timeout=timeout)
+    def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
+        logger.info("Running rails command `{command}`".format(command=command))
+        return self.run_command('/var/www/miq/vmdb/bin/rails runner {command}'.format(
+            command=command), timeout=timeout, **kwargs)
 
-    def run_rake_command(self, command, timeout=RUNCMD_TIMEOUT):
-        logger.info("Running rake command `%s`", command)
-        return self.run_command('cd /var/www/miq/vmdb; bin/rake {}'.format(command),
-            timeout=timeout)
+    def run_rake_command(self, command, timeout=RUNCMD_TIMEOUT, **kwargs):
+        logger.info("Running rake command `{command}`".format(command=command))
+        return self.run_command(
+            '/var/www/miq/vmdb/bin/rake -f /var/www/miq/vmdb/Rakefile {command}'.format(
```

Comment:
@psav, same with rake command, removed "cd" here and using absolute path. and referencing Rakefile with -f option.